lib/matplotlib/tests/test_subplots.py

```diff
@@ -142,14 +142,6 @@ def test_shared_and_moved():
     check_visible([a1], [False], [True])
 
 
-def test_exceptions():
-    # TODO should this test more options?
-    with pytest.raises(ValueError):
-        plt.subplots(2, 2, sharex='blah')
-    with pytest.raises(ValueError):
-        plt.subplots(2, 2, sharey='blah')
```

Comment:
These tests should remain. They are still valid under this PR, and it's reasonable to not allow arbitrary strings being accepted because that would allow typos to slip through.

Reply:
I agree, this test should not be removed.